app/views/homes/index.html.erb

```diff
@@ -1 +1,2 @@
 <div id="app"></div>
+<%= link_to "Daily Buzz Home Page", coffee_shops_path %>
```

Comment:
This link should actually either be in `application.html.erb` (which would mean it would appear on every page, react included 

Or, if only appearing in React pages, in a Layout.js component that wraps around all of your other components (see React Router reading) 

At the moment, looks like this will only appear on React pages, so for a better separation of concerns, I would move it to React